content/ja/docs/community/history.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "プロジェクトの歴史"
+description: "プロジェクトの歴史の概要を提供します。"
+aliases: ["/docs/history/"]
+weight: 4
+---
+
+Helm 3 は、
+[インキュベーションの最終段階](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc) の
+[CNCF プロジェクト](https://www.cncf.io/projects/) です。
```

Comment:
Please update this to say "Helm is a [graduated](https://helm.sh/blog/celebrating-helms-cncf-graduation/)
[CNCF project](https://www.cncf.io/projects/)." See https://github.com/helm/helm-www/pull/767/files.